src/ray/common/buffer.h

```diff
@@ -1,3 +1,17 @@
+// Copyright 2020 The Ray Authors.
```

Comment:
Any idea what date this should be? Should it be the project start or something else?

Reply:
I have seen people update it in the LICENSE file every year, and in the source files just put the year that the file was checked in.

Why don't we put the year as `2017` (when the project was open sourced) for all files, to simplify things? It would be too much work to check for each file when it was added. This way we can just do search and replace on the PR.

Reply:
While we are at it, let's also add "All rights reserved.", i.e.
```
// Copyright 2017 The Ray Authors. All rights reserved.
```

Reply:
Actually, that doesn't seem to be in the recommendation: http://www.apache.org/licenses/LICENSE-2.0.html#apply